### DISCUSSION TITLE
How to integrate new echart into Apache Superset?

### DISCUSSION THREAD

**Discussion by GaryMam** [apache/superset#16237] [Q&A / Help]
Can you please let me know the steps to integrate new echarts into Apache supserset? I am very new to this and online is not giving much information about it. I am currently using MacOs.

**Comment by junlincc** [comment]
This is a board question. Are you a frontend developer? which Echarts are you thinking?

**Reply by JayiShen** [reply]
Hi Junlin -  Are individual front-end developers still be able to integrate e-charts based on our own needs? Or it's gonna be done by Superset via each release? Thx.
